python-sdk/docs/guides/databricks.rst

```diff
@@ -0,0 +1,91 @@
+.. _databricks:
+
+======================
+Databricks Integration
+======================
+Introduction
+
+The Astro Python SDK now allows users to easily access and work with data stored in Databricks through the Delta API. The Astro Python SDK provides a simple, yet powerful interface for working with Spark and Delta tables in Databricks, allowing users to take advantage of the scalability and performance of Databricks while maintaining the flexibility of Airflow DAG authoring and all the benefits of Airflow scheduling.
```

Comment:
should we just link out to Delta API too - https://docs.delta.io/latest/api/python/index.html

Reply:
Can you also split this into multiple lines rather than a single line, please